include/zephyr/drivers/clock_control/clock_control_tccvcp.h

```diff
@@ -0,0 +1,310 @@
+/*
+ * Copyright 2024 Hounjoung Rim <hounjoung@tsnlab.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+#ifndef CLOCK_CONTROL_TCCVCP_HEADER
+#define CLOCK_CONTROL_TCCVCP_HEADER
+
+#define CLOCK_PLL_MAX_NUM (2)
+#define CLOCK_SRC_MAX_NUM (6) /*  ((CLOCK_PLL_MAX_NUM * 2 ) + 2) */
```

Comment:
주석으로 달지 말고 그냥 직접 넣어도 되지 않을까요?
```suggestion
#define CLOCK_SRC_MAX_NUM ((CLOCK_PLL_MAX_NUM * 2 ) + 2)
```